APMrover2/APMrover2.cpp

```diff
@@ -75,9 +75,19 @@ const AP_Scheduler::Task Rover::scheduler_tasks[] = {
     SCHED_TASK(accel_cal_update,       10,    100),
     SCHED_TASK(dataflash_periodic,     50,    300),
     SCHED_TASK(button_update,          5,     100),
+    SCHED_TASK(stats_update,           1,     100),
 };
 
 /*
+  update AP_Stats
+*/
+void Rover::stats_update(void)
+{
+    g2.stats.set_flying(motor_active());
```

Comment:
perhaps set_flying() should be a more generic term across the platforms such as "set_run_timer" or set_timer_active or set_is_moving. You'll be setting the same thing in ArduSub and for antenna tracker you'll want to know the age of the servos.